public/index.html

```diff
@@ -0,0 +1,24 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Product Options</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <div class="appWrapper">
+      <div class="storePath">Target / Men / Men's Clothing / Hoodies & Sweatshirts</div>
+      <div class="itemNameHeader">Men's Standard Fit French Terry Hoodie Sweatshirt - Goodfellow & Co™</div>
+      <div class="ShopAll">Shop all Goodfellow & Co</div>
```

Comment:
are these 3 div's necessary? Seems they might be carry-overs from productOptions

Reply:
Those add the title and other stuff at the top
<img width="672" alt="Screen Shot 2020-10-07 at 9 39 00 PM" src="https://user-images.githubusercontent.com/37489716/95373310-83ff1f00-08e5-11eb-87fb-2f6023fdd1bb.png">

Reply:
Got it. Makes sense.